.travis.yml

```diff
@@ -5,8 +5,8 @@ language: go
 go_import_path: github.com/coreos/ignition
 
 go:
-  - 1.8
-  - 1.9
+  - "1.10.4"
```

Comment:
As I don't know what we'll be using to build ignition on RHCOS and FCOS, I'd suggest to stick here to a reasonable minimum version (1.8.x seemed fine) and only use the other entries to track latest stable Go. (Unless there is some specific breakage on 1.8/1.9 and we are requiring >= 1.10 now).

Reply:
SGTM, fixed.